Sources/SwiftFormat/Rules/AlwaysUseLowerCamelCase.swift

```diff
@@ -100,7 +100,11 @@ public final class AlwaysUseLowerCamelCase: SyntaxLintRule {
 
     // We allow underscores in test names, because there's an existing convention of using
     // underscores to separate phrases in very detailed test names.
-    let allowUnderscores = testCaseFuncs.contains(node)
+    let allowUnderscores = testCaseFuncs.contains(node) || node.attributes.contains {
```

Comment:
It would be great to factor this new check out into a convenience method on `WithAttributesSyntax`, which could then be used elsewhere (and on any declaration that can have attributes). What about something like this?

```
extension WithAttributesSyntax {
  func hasAttribute(_ name: String) { ... }
}
```

You could create a new `WithAttributesSyntax+Convenience.swift` file in Sources/SwiftFormat/Core for this (make sure to also add the new file to the CMakeLists.txt file in Sources/SwiftFormat).

If you're feeing really bold, you could also support the fully-qualified version: `@Testing.Test`.

Reply:
That's a good idea, will do it together with adding Swift Testing support for `NeverForceUnwrap`,  `NeverUseForceTry` and `NeverUseImplicitlyUnwrappedOptionals` rules to this PR. 
Thanks